src/main/java/FigureExapmple/Color.java

```diff
@@ -0,0 +1,21 @@
+package FigureExapmple;
+
+public enum Color {
+    BLUE (" синий."),
```

Comment:
```suggestion
    BLUE ("синий"),
```
в такому випадку ми маємо більш універсальний єнам